packages/account/src/Components/forms/personal-details-form.jsx

```diff
@@ -751,14 +751,7 @@ const TaxResidenceField = ({
     );
 };
 
-const TaxIdentificationNumberField = ({
-    is_tin_popover_open,
-    setIsTinPopoverOpen,
-    setIsTaxResidencePopoverOpen,
-    disabled,
-    required = false,
-}) => {
-    const { isDesktop } = useDevice();
+const TaxIdentificationNumberField = ({ disabled, required = false }) => {
     return (
```

Comment:
cant we remove states and props related to is_tin_popover_open from parent component

Reply:
Done